HLTrigger/special/plugins/HLTMuonPointingFilter.cc

```diff
@@ -39,6 +39,9 @@ HLTMuonPointingFilter::HLTMuonPointingFilter(const edm::ParameterSet& pset) :
   thePropagatorName(pset.getParameter<std::string>("PropagatorName") ),
   theRadius(        pset.getParameter<double>("radius") ),              // cyl's radius (cm)
   theMaxZ(          pset.getParameter<double>("maxZ") ),                // cyl's half lenght (cm)
+  thenPixHits(      pset.getParameter<unsigned int>("nPixHits") ),      // pixel hits
+  theTkLayers(      pset.getParameter<unsigned int>("TkLayers") ),      // tracker layers with measurements
+  thenMuonHits(     pset.getParameter<unsigned int>("nMuonHits") ),     // muon hits
   thePropagator(nullptr),
```

Comment:
The new variables are all integer numbers, counting something.
Why are two of them called nSomething, while the third is no 'n'?
I also find name thenMuonHits confusing, why 'then' :)?
I suggest theNmuonHits, or theMuonHits, but at least make it consistent between the variables.